packages/xml-views-validation/test/validators/element/type-of-aggregation-spec.ts

```diff
@@ -0,0 +1,97 @@
+import { partial } from "lodash";
```

Comment:
the pure xml snippets may no be enough to understand why a test should fail or succeed. Basically we do not know from the XML what are the expected and actual types of the aggregation/class.

Perhaps each test should contain an XML Comment in its snippet
to add context to the scenarios